src/meshdb/templates/admin/password_reset/password_reset_complete.html

```diff
@@ -1,3 +0,0 @@
-{% extends "registration/password_reset_complete.html" %}
-
-{% block map_sidebar %}{% endblock %}
```

Comment:
Hmm did you confirm these pages still work correctly and look good? I think the reason we had this before is that the map didn't look right on them. Also does this mean we show the map on the login page? I don't have a particular problem with that but it feels a bit odd

Reply:
Whoops, thanks. I put those back. Login page doesn't have the map on it anymore, but there's currently a bug where the logout page does. I have a hamfisted idea to fix that.

Reply:
fixed tm